pymc/distributions/continuous.py

```diff
@@ -152,7 +152,7 @@ class BoundedContinuous(Continuous):
     """Base class for bounded continuous distributions"""
 
     # Indices of the arguments that define the lower and upper bounds of the distribution
-    bound_args_indices: list[int] | None = None
+    bound_args_indices: tuple[int, int] | None = None
```

Comment:
Aren't there distributions with only one of the bounds?

Reply:
You're right, [there are](https://github.com/thomasaarholt/pymc/blob/157c29a60edbe670e245b1208e3e97749669da8d/pymc/distributions/continuous.py#L2084)! I was wondering why mypy didn't flag it, but of course it was originally an error as `list[int]` doesn't allow `(3, None)`! Fixing this